docs/ossm/cert-manager/README.md

```diff
@@ -0,0 +1,200 @@
+[Return to OSSM Docs](../)
+
+# Cert Manager and istio-csr Integration
+
+Below are instructions for integrating cert-manager with OpenShift Service Mesh 3. It largely follows the [cert-manager istio-csr documentation](https://cert-manager.io/docs/usage/istio-csr/) but you will need to adjust a few settings based on which `updateStrategy` you are using.
+
+## Common Setup
+
+These steps are the same for each `updateStrategy`.
+
+1. Install the [cert-manager Operator for Red Hat OpenShift](https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/security_and_compliance/cert-manager-operator-for-red-hat-openshift#cert-manager-operator-install).
+
+2. Create the `istio-system` namespace for the root cert and for your `Istio` resource.
+
+   ```sh
+   oc create namespace istio-system
+   ```
+
+3. Create a Self Signed certificate. Note you should adapt this example based on what PKI you are using e.g. using a vault `Issuer` instead of a self signed `Issuer`.
+
+   ```sh
+   oc apply -f https://raw.githubusercontent.com/cert-manager/website/7f5b2be9dd67831574b9bde2407bed4a920b691c/content/docs/tutorials/istio-csr/example/example-issuer.yaml
+   ```
+
+4. Export the root CA to the `cert-manager` namespace.
+
+   ```sh
+   oc get -n istio-system secret istio-ca -o jsonpath='{.data.tls\.crt}' | base64 -d > ca.pem
+   oc create secret generic -n cert-manager istio-root-ca --from-file=ca.pem=ca.pem
+   ```
+
+## InPlace Strategy
+
```

Comment:
For clarity: do users perform the Common Setup first, and then these steps second if they are following the `InPlace` strategy?

Reply:
Yep